modules/navier_stokes/src/postprocessors/VolumetricFlowRate.C

```diff
@@ -116,7 +116,7 @@ VolumetricFlowRate::computeQpIntegral()
     {
       // FIXME The material property would need to be computed on the face #16809
       // This requires knowing the neighbor value, (1)MaterialInterface cant do that
-      advected_quantity = MetaPhysicL::raw_value(_advected_material_property[_qp]);
+      advected_quantity = MetaPhysicL::raw_value(_advected_material_property(_qp));
```

Comment:
Ok I will make this change as well as for the one above. I guess this shouldn't require re-golding actually 🤔

Reply:
Probably will need a regold. We should hit 300 on the spot for inlet value and we were like a little off because of the lack of material face values

Reply:
In order to get this postprocessor "right" it needs to use the same flux computation strategy as the fv flux kernel or bc that computes the flux. Right now this postprocessor is not doing any interpolation on the boundary like the corresponding flux kernels are doing. I think this should be a change that happens in a follow-on PR because it is more than a simple functor material property substitution.

Reply:
Because one could reasonably argue that it is the flux kernel computation of the advected quantity on the boundary that needs to change as opposed or addition to changing this postprocessor

Reply:
I agree with all that and this is a bigger problem for the variable. 
I m ok with no fix there instead of a partial fix of just the material property evaluated the same way as the kernel